docs/_articles/en/architecture/retire-js-engine.md

```diff
@@ -0,0 +1,23 @@
+---
+title: 'RetireJS'
+lang: en
+---
+## What is RetireJS?
+[RetireJS](https://retirejs.github.io/retire.js/) is an engine that analyzes a project's third-party dependencies and
```

Comment:
Would adding Javascript be redundant? i.e. `third-party Javascript dependencies`